loopchain/baseservice/rest_client.py

```diff
@@ -168,7 +169,7 @@ def _call_jsonrpc(self, target: str, method: RestMethod, params: Optional[NamedT
         # 'aioHttpClient' does not support 'timeout'
         url = self._create_jsonrpc_url(target, method)
         async with ClientSession() as session:
-            http_client = aiohttpClient(session, url)
+            http_client = AiohttpClient(session, url)
```

Comment:
Can you put `timeout` as a parameter?
As you can see an comment above, the old version JsonRPCClient does not support timeout parameter. But this one seems to support it.

Reply:
Added. 
Please check this commit and resolve if it is acceptable.

https://github.com/icon-project/loopchain/pull/476/commits/540346d7ad68e846819c2dd14d57930158f7ce5d